root@conforti2:/home/ubuntu/master# python3 serverCoAP.py 
PRE-DUMP size: 29M	/runc/containers/python_bundle/parent

Number of files: 4 (reg: 3, dir: 1)
Number of created files: 4 (reg: 3, dir: 1)
Number of deleted files: 0
Number of regular files transferred: 3
Total file size: 29.86M bytes
Total transferred file size: 29.86M bytes
Literal data: 29.86M bytes
Matched data: 0 bytes
File list size: 0
File list generation time: 0.001 seconds
File list transfer time: 0.000 seconds
Total bytes sent: 6.53M
Total bytes received: 77

sent 6.53M bytes  received 77 bytes  2.61M bytes/sec
total size is 29.86M  speedup is 4.57
Transferring PRE-DUMP to 172.16.4.4
time -p runc checkpoint --image-path image  --parent-path ../parent --lazy-pages --page-server localhost:27 --status-fd /tmp/postcopy-pipe python_bundle
Ready for lazy page transfer
time -p runc checkpoint --image-path image  --parent-path ../parent --lazy-pages --page-server localhost:27 --status-fd /tmp/postcopy-pipe python_bundle finished after 0.48 second(s) with 0
DUMP size: 312K	/runc/containers/python_bundle/image

Number of files: 65 (reg: 63, dir: 1, link: 1)
Number of created files: 65 (reg: 63, dir: 1, link: 1)
Number of deleted files: 0
Number of regular files transferred: 63
Total file size: 123.27K bytes
Total transferred file size: 123.26K bytes
Literal data: 123.26K bytes
Matched data: 0 bytes
File list size: 0
File list generation time: 0.001 seconds
File list transfer time: 0.000 seconds
Total bytes sent: 40.84K
Total bytes received: 1.22K

sent 40.84K bytes  received 1.22K bytes  84.12K bytes/sec
total size is 123.27K  speedup is 2.93
Transferring DUMP to 172.16.4.4
runc restored python_bundle successfully
real 0.17
user 0.02
sys 0.05
real 1.84
user 1.11
sys 0.03
real 0.81
user 0.04
sys 0.00
real 1.69
user 0.04
sys 0.09
Something did not work. Exiting!
time="2021-09-19T03:42:43Z" level=error msg="container \"python_bundle\" does not exist"
container "python_bundle" does not exist
Command exited with non-zero status 1
real 0.01
user 0.00
sys 0.01